ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/response/model/HeaderRecordInfo.java

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 *  Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org).
```

Reply:
Why are we removing this?

Reply:
Had an offline discussion and discussed to address this in this issue: https://github.com/ballerina-platform/ballerina-library/issues/6043